initialization/initialization_test.go

```diff
@@ -77,6 +76,29 @@ func TestInitialize(t *testing.T) {
 	r.NoError(verifying.VerifyVRFNonce(init.Nonce(), m))
 }
 
+func TestMaxFileSize(t *testing.T) {
+	r := require.New(t)
+
+	cfg := Config{}
+	cfg.BitsPerLabel = 8
+	cfg.LabelsPerUnit = 2048
```

Comment:
Nit, how about initializing in-place rather than creating and mutating?
```suggestion
	cfg := Config{
		BitsPerLabel: 8,
		LabelsPerUnit : 2048,
	}
```

Reply:
Fixed.